api/python/provisioner/commands/__init__.py

```diff
@@ -1236,6 +1245,13 @@ def _passwordless_ssh():
                           '/usr/bin/provisioner, '
                           '/var/log, '
                           '/tmp, '
+                          '/usr/bin/lsscsi, '
+                          '/usr/sbin/mdadm, '
+                          '/usr/sbin/sfdisk, '
+                          '/usr/sbin/mkfs, '
+                          '/usr/bin/rsync, '
+                          '/bin/rsync, '
+                          '/usr/local/bin/rsync, '
```

Comment:
`/usr/local/bin/rsync` not requried.
rsync installation is using yum. The binary would land in `/usr/bin` only.

Reply:
I agree. Just for better understanding: all these issues with `/usr/local/bin` that we faced before were about pip vs yum ways of installation:
-  pip installed tool to /usr/local/bin
- yum installed to /usr/bin

For now it might be ok to make our life safer and cover both cases for salt but let's not abuse. We need a very strong concerns for any extensions here.